addons/google_calendar/models/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+
+import google_calendar
+import calendar_attendee
+import calendar_event
+import res_users
+import res_config
```

Comment:
Its not needed but convenient for good code, Imports arrange alphabetically 
Like : 
```
import calendar_attendee
import calendar_event
import google_calendar
import res_config
import res_users
```